enclave/crypto/openssl/init.c

```diff
@@ -15,6 +14,24 @@ int _is_symcrypt_engine_available = 0;
 int SCOSSL_ENGINE_Initialize();
 int oe_sgx_get_additional_host_entropy(uint8_t*, size_t);
 
+static int _initialize_symcrypt_engine()
+{
+    int result;
+
+    result = SCOSSL_ENGINE_Initialize();
+    if (result != OE_SYMCRYPT_ENGINE_SUCCESS)
+        goto done;
+
+done:
+    if (result == OE_SYMCRYPT_ENGINE_FAIL)
```

Comment:
Can result be a value other than `OE_SYMCRYPT_ENGINE_FAIL` or `OE_SYMCRYPT_ENGINE_SUCCESS`? If so, error message won't always be printed.